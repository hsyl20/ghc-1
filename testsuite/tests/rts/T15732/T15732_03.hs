import GHC.RTS.Flags ( getGCFlags, GCFlags(..) )

main :: IO ()
main = do
  gcflags <- getGCFlags
  putStrLn $ "oldGenFactor: " ++ show (oldGenFactor gcflags)
  putStrLn $ "generations: " ++ show (generations gcflags)
