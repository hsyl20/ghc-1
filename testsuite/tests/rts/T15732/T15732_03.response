+RTS @T15732_03a.response -RTS
