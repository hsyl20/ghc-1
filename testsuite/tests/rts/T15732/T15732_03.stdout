oldGenFactor: 3.0
generations: 4
